code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -5,8 +5,16 @@
 	icon = 'icons/mob/alien.dmi'
 	icon_state = "larva0_dead"
 	food_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/toxin/acid = 10)
-	var/stage = 0
+	//Which growth stage are we currently on? Stages give hints to the organ holder that this is in them.
+	var/stage = -1	//Once we initialise, we call our first growth cycle which immediately puts us to stage 0.
```

Comment:
Either make it start at zero and increase the max stage value by one, or make this a define.